tests/Bridge/Doctrine/Orm/SubresourceDataProviderTest.php

```diff
@@ -267,6 +268,64 @@ public function testGetSubSubresourceItem()
         $this->assertEquals($result, $dataProvider->getSubresource(ThirdLevel::class, ['id' => ['id' => 1], 'relatedDummies' => ['id' => 1]], $context));
     }
 
+    public function testGetSubresourceOneToOneOwningRelation()
+    {
+        // RelatedOwningDummy OneToOne Dummy
+        $dql = 'SELECT ownedDummy_a2 FROM ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\Dummy id_a1 INNER JOIN id_a1.ownedDummy ownedDummy_a2 WHERE id_a1.id = :id_p1';
+
+        $queryProphecy = $this->prophesize(AbstractQuery::class);
+        $queryProphecy->getOneOrNullResult()->shouldBeCalled()->willReturn([]);
+
+        $identifiers = ['id'];
+        $queryBuilder = $this->prophesize(QueryBuilder::class);
+        $queryBuilder->setParameter('id_p1', 1, DBALType::INTEGER)->shouldBeCalled()->willReturn($queryBuilder);
+        $funcProphecy = $this->prophesize(Func::class);
+        $func = $funcProphecy->reveal();
+
+        $queryBuilder->andWhere($func)->shouldBeCalled()->willReturn($queryBuilder);
+
+        $queryBuilder->getQuery()->shouldBeCalled()->willReturn($queryProphecy->reveal());
```

Comment:
Nice, I was going to said that you should use the same ManagerRegistry for both and that's what you did so :+1:. You can remove the blank lines `284` and `286` but looks good to me.

Reply:
done!